Add Merge Action Groups section to the MFTF guide (#3792)
magento/devdocs: mftf/2.3/merging.md

_This topic was updated due to the {{page.mftf-release}} MFTF release._
{: style="text-align: right"}

The MFTF allows you to merge test components defined in XML files, such as:

> - [tests]

Reviewer: Are these called "tests", "sections", etc. in the code? If so they should be in code formatting, encased with backticks. If not, consider capitalizing them since they are the first word in a bullet point.

Author: Fixed.

</test>
```

## Merge action groups

Merging action groups allows you to extend existing tests by reusing existing action groups, while customizing them for your specific needs.

> ### Use Case

Reviewer: ```suggestion
### Use case
```

> Here is an action group for selecting `customerGroup` in the `Cart Price Rules` section.

Reviewer: ```suggestion
The following example is an action group for selecting `customerGroup` in the `Cart Price Rules` section.
```

Author: Grammar fix.

> The controls change drastically in the B2B version, so it was abstracted to an action group.

Reviewer: ```suggestion
The controls change drastically in the B2B version, so it was abstracted to an action group so that it may be easily changed if B2B is enabled. 
```

> This way it could be easily changed if B2B is enabled.

Reviewer: ```suggestion
```

Author: Done.

> 

Reviewer: ```suggestion
> Action group for selecting `customerGroup` in the `Cart Price Rules` section
```

Author: Thanks.

```xml
<actionGroup name="selectNotLoggedInCustomerGroup">
    <selectOption selector="{{AdminCartPriceRulesFormSection.customerGroups}}" userInput="NOT LOGGED IN" stepKey="selectCustomerGroup"/>
</actionGroup>

> B2B Merge file

Reviewer: ```suggestion
> B2B merge file
```

Reviewer: The > acts as a subtitle of sorts for the code sample.

Author: Nice!
